src/framework/theme/components/route-tabset/route-tabset.component.ts

```diff
@@ -102,6 +102,12 @@ export class NbRouteTabsetComponent {
    */
   @Input() tabs: any[];
 
+  /**
+   * Options passed to `routerLinkActiveOptions` directive which set on tab links.
+   * `{ exact: true }` by default.
+   */
+  @Input() activeLinkOptions = { exact: true };
```

Comment:
should we add :any or better LinkOptions type here? As = { exact: true } matches activeLinkOptions variable to the exact type.